src/commands/indexer/index.ts

```diff
@@ -548,23 +547,15 @@ export default class Indexer extends HealthCheck {
       }
     }
 
-    await (isNewMint
-      ? handleTransferERC721Event.call(
-          this,
-          this.networkMonitor,
-          interestingTransaction.transaction,
-          job.network,
-          event,
-          tags,
-        )
-      : handleTransferEvent.call(
-          this,
-          this.networkMonitor,
-          interestingTransaction.transaction,
-          job.network,
-          event,
-          tags,
-        ))
+    handleTransferERC721Event.call(
```

Comment:
@BravoNatalie So i know we have the `this.handleTransferERC721Event` function and the file `handleTransferERC721Event` function. I want to get rid of all the small wrapper functions and just call the file version instead. 

update the code so that all of the logic is being done in the file `handleTransferERC721Event` and remove the `this.handleTransferERC721Event` function. 

In another ticket we can have you remove the other wrapper calls instead.